styx-common/src/main/java/com/spotify/styx/monitoring/MetricsStats.java

```diff
@@ -377,4 +397,9 @@ private Histogram tickHistogram(String type) {
     return tickHistograms.computeIfAbsent(
         type, (op) -> registry.histogram(TICK_DURATION.tagged("type", type)));
   }
+
+  private Meter datastoreOperationMeter(String operation, String kind) {
+    return datastoreOperationMeters.computeIfAbsent(Tuple.of(operation, kind),
+        t -> registry.meter(DATASTORE_OPERATION_RATE.tagged("operation", operation, "kind", kind)));
```

Comment:
align the naming convertion? s/t/(op)/

Reply:
But it's a tuple of operation and kind. And it's not used by the lambda. I would use `_` if I could but that's not allowed anymore/yet.

Reply:
Ok ignore this comment.